pipenv/routines/clean.py

```diff
@@ -2,6 +2,7 @@
 
 from pipenv.patched.pip._internal.build_env import get_runnable_pip
 from pipenv.routines.lock import do_lock
+from pipenv.utils import console, err
```

Comment:
But where does this import come from?

Reply:
I initialized them here:
https://github.com/pypa/pipenv/blob/8240ea73d2e348960f97ca01e7ed0f169876fb0c/pipenv/utils/__init__.py